src/main/kotlin/com/atlassian/performance/tools/ssh/api/SshConnection.kt

```diff
@@ -153,6 +153,12 @@ interface SshConnection : Closeable {
         remoteDestination: String
     )
 
+    /**
+     * @since 2.3.0
+     */
+    @JvmDefault
+    fun getHost(): SshHost = throw Exception("Not implemented")
```

Comment:
Does it mean we need to use special compilation flag for this to work?

Reply:
Yes

Reply:
Should we document it somehow? somewhere?

Reply:
Note that this extra dependency was added back in 2.0.0, so we're not breaking anything.
If there is any working implementation of this interface it already has this flag or doesn't need it (e.g. in Java).

Reply:
Ok I see it: https://github.com/atlassian/ssh/commit/a0b8a6b14336e5c1c46c652ecd4caaf103d26bf6